### DISCUSSION TITLE
如何实现智能扫描定制化开发

### DISCUSSION THREAD

**Discussion by Harryysw** [actiontech/sqle#1678] [Q&A]
SQLE中具备【智能扫描】功能，该功能不同的数据源内置了一些智能扫描任务模板。
如果想对某数据源定制开发一个智能扫描任务模板，应该如何实现？数据库审核插件可以实现吗？

**Comment by sjjian** [comment]
https://actiontech.github.io/sqle-docs-cn/3.modules/3.6_auditplan/scanner_management.html， 自己定制的话原理很简单的，无非就是要通过接口将SQL推送到SQLE，代码参考：SQL采集：https://github.com/actiontech/sqle/blob/main/sqle/cmd/scannerd/cmd/mybatis.go ；定义扫描任务：https://github.com/actiontech/sqle/blob/main/sqle/server/auditplan/meta.go

**Comment by Harryysw** [comment]
邮件已收到~
